app/controllers/vm_infra_controller.rb

```diff
@@ -73,4 +73,16 @@ def skip_breadcrumb?
 
   menu_section :inf
   has_custom_buttons
+
+  def vm_transform
+    vm = Vm.find_by(:id => params[:id].to_i)
```

Comment:
Oh, and this should be `vm = find_records_with_rbac(Vm, [params[:id].to_i]).first`.

(Though not sure if it wouldn't be better to fix in a separate PR, since that function is not available in fine.)